static/src/stylesheets/module/survey/_survey-page.scss

```diff
@@ -0,0 +1,71 @@
+//Styles to be used on pages that contain full formstack surveys
+
+.formstack-survey-wrapper {
+    @include gs-container;
+
+    @include mq($until: tablet) {
+        padding: $gs-gutter;
+    }
+
+    h1 {
+        @include fs-headline(4);
+        color: $guardian-brand;
+        font-weight: bold;
+        margin-top: $gs-baseline/2;
+        margin-bottom: $gs-baseline;
+
+        @include mq(tablet) {
+            @include fs-headline(6);
+            font-weight: bold;
+        }
+    }
+
+    h2 {
+        @include fs-header(4);
+        color: $guardian-brand;
+        margin-bottom: $gs-baseline/2;
+    }
+
+    p {
+        @include fs-bodyCopy(2, size-only = true);
+        margin-bottom: $gs-baseline;
+    }
+
+    .fsError:before {
+        content: '';
+        display: block;
+        background: $error-icon;
+        width: 18px;
+        height: 18px;
+        float: left;
+        margin: 0 .5em 0 0;
+    }
+
+    .fsError {
+        color: $error;
+        background-color: unset;
+        border: 0;
+        text-align: left;
+        padding-left: 0;
+    }
+
+    .fsSubmitButton {
+        @include fs-textSans(2);
```

Comment:
We can't remove the `!important` rules as lots of rules come from form stack and I couldn't see an easy way to override them other than `important`.